docs/usage/config.md

```diff
@@ -95,24 +95,27 @@ Please take note of the fact that for a different reference fasta.gz the  unzipp
+| vcf.annotate.vep_plugin_dir                     | *installed* |                                                                                                                                                                                                                                                             |
+| vcf.annotate.vep_plugin_hpo                     | *installed* |                                                                                                                                                                                                                                                             |
+| vcf.annotate.vep_plugin_inheritance             | *installed* |                                                                                                                                                                                                                                                             |
+| vcf.annotate.vep_plugin_vkgl_mode               | 1           | allowed values: [0=full VKGL, 1=public VKGL]. update `vcf.annotate.GRCh38.vep_plugin_vkgl` accordingly                                                                                                                                                      |
+| vcf.annotate.GRCh38.capice_model                | *installed* |                                                                                                                                                                                                                                                             |
+| vcf.annotate.GRCh38.vep_custom_phylop           | *installed* |                                                                                                                                                                                                                                                             |
+| vcf.annotate.GRCh38.vep_plugin_clinvar          | *installed* |                                                                                                                                                                                                                                                             |
+| vcf.annotate.GRCh38.vep_plugin_gnomad           | *installed* |                                                                                                                                                                                                                                                             |
+| vcf.annotate.GRCh38.vep_plugin_green_db_enabled | true        |                                                                                                                                                                                                                                                             |
```

Comment:
1) default: `false` instead of `true`
2) description: enabling is only allowed for academic use, for details see [here](https://doi.org/10.5281/zenodo.5636209)